taxonomy/utils.py

```diff
@@ -53,51 +53,56 @@ def get_whitelisted_serialized_skills(course_key):
     return skills_data
 
 
-def update_course_skills_data(course_key, skill_external_id, confidence, skill_data):
+def get_product_identifier(product_type):
     """
-    Persist the course skills data in the database.
+    Return the identifier of a Product Model.
     """
-    skill, __ = Skill.objects.update_or_create(external_id=skill_external_id, defaults=skill_data)
+    identifier = None
+    if product_type == 'Program':
+        identifier = 'uuid'
```

Comment:
nit: Please add a comment that this UUID is the value from Discovery and not program_uuid in ProgramSkills.